app/Http/Controllers/ProjectController.php

```diff
@@ -51,16 +52,17 @@ public function index(Request $request)
      */
     public function store(Request $request)
     {
-        $project = new Project([
-          'name' => $request->get('name'),
-          'information' => $request->get('information'),
-          'deadline' => $request->get('deadline'),
-          'type' => $request->get('type'),
-          'status' => $request->get('status')
+      $request->validate(self::RULES);
+      $project = new Project([
+        'name' => $request->get('name'),
```

Comment:
Your indentation is still broken on these lines. Please indent using 4 spaces.

Reply:
i will change to 4 at next pull request